src/tiles/grid/tile-stats.ts

```diff
@@ -93,14 +93,15 @@ export class TileStats extends LitElement {
 
       .item-stats {
         height: 35px;
+        padding-left: 5px;
+        padding-right: 5px;
       }
 
       #stats-row {
-        border-top: 1px solid #bbb;
-        align-items: center;
+        align-items: left;
```

Comment:
I don't think `align-items: left` is a valid rule here. Did you mean `text-align: left`?

Is this meant to adjust the mediatype icon positioning? I notice they seem a bit misaligned so maybe this will fix it? 😄

Reply:
Oh, I see now there's already a `text-align: center` rule below, which seems correct. And now I'm finding the mediatype icons are aligned properly in the demo app, so I probably just messed something up while testing. Oops! 🙃 

So this should probably still be `align-items: center` like before, right?

Reply:
i removed `align-items: left` since ideally `text-align: center` should align the numbers stats text